phac-nml/irida-next
# Samples Table: Fixes preserving page size & number when toggling metadata

## What does this PR do and why?
Persisting the page size & number during metadata toggling.
Fixes #737.

## Screenshots or screen recordings
![image](https://github.com/user-attachments/assets/9654cb92-7457-45c8-a6be-e96660770e13)
![image](https://github.com/user-attachments/assets/300f11fb-3506-4547-85b3-751bc9074b6b)

## How to set up and validate locally
1. Navigate to a group or project samples page.
2. Select a page size other than the default.
3. Toggle the metadata button.
4. Verify the page size & number remain unchanged.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
